### PR TITLE
Details modal

### PR DESCRIPTION
**Pull request by Wikitelmex**
# Modal implemented
- Add modal
- styling modal
- project background blur and darken when modal appears
- buttons with icons
- responsive modal
- modal appears when click the "see project" button inside of multi-post stories card
- modal load via JS with array data
- cards load via JS with array data
- styling corrections and improvements
- deletion of static cards

### CODE REVIEW

**Comment by mikenath223** [comment]
Hi , 

You made a great effort in this project, but unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that will be invalidated for this reason:
-  Kindly add a description to the PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding. Cheers!🚀🚀

If something isn't 100% clear, you can leave comments or questions in this PR thread.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **don't count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -34,271 +35,308 @@
...
+                                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                <ul class="tags">
+                                    <li class="card-tag">html</li>
+                                    <li class="card-tag">bootstrap</li>
+                                    <li class="card-tag">ruby</li>
+                                </ul>
+                            </div>
+                            <div class="card-footer">
+                                <a href="https://wikitelmex.github.io/Portfolio/">See Project</a>
+                            </div>
```
Comment (DanteAlonsoHT) [inline index.html]: This part of your code should be generated dynamically using JavaScript, you should create a constant like this:
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
Note: With all of your projects, and then you should display these projects dynamically using JavaScript, you can use the "forEach()" method and "innerHTML" to do it 😉 

**Feel free to check the example of students projects:**
[Example 1](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)
[Example 2](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96)

- [x] Your web app should be dynamically displaying the projects.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @Wikitelmex, 

Amazing job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @Wikitelmex , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**